### PR TITLE
Add instructions for Tech Portfolio steps for CSP account and user creation

### PR DESCRIPTION
**Pull request by JJediny**
All the specific steps for each CSP will be made into their own issue templates

### CODE REVIEW

File: how_we_work/account_management/cloud-service-providers.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Cloud Service Provider (CSP) Account Management Playbook
+
+The TTS Tech Portfolio provisions and manages our Infrastructure-as-a-Service (IaaS) accounts.
+
+TTS employees request new accounts or access to existing account using this google form - https://docs.google.com/forms/d/1SiD7E6hGV3eq9MYqg9na0A5fepG_UPEQWSYBNMPspcY/edit
+
+Form responses are tracked here - https://docs.google.com/spreadsheets/d/1TdFKjzy1B_eHyTjKwWb90As-wcwaU8rYBlk3Y_TiNoU/edit?resourcekey#gid=1269445049
+
+::: info
+Tech Portfolio members should sign up for notifications to changes in the sheet.
```
Comment (adborden) [inline how_we_work/account_management/cloud-service-providers.md:10]: This should be part of the onboarding checklist.

File: how_we_work/account_management/cloud-service-providers.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+::: 
+
+When a new request is made the Tech Portfolio should immediately acknowledge the request by reaching out to the requestor via slack, email, or gchat.
```
Comment (adborden) [inline how_we_work/account_management/cloud-service-providers.md:13]: Should this be a responsibility for Ops Rotation?

**Review by adborden — CHANGES_REQUESTED** [review]
Let's make sure this is part of the onboarding checklist, otherwise this seems like team members won't be notified of new requests.

How about this lives in the TTS Handbook? Similar to how we administer any other tool, I think having admin docs side-by-side with user docs is helpful and more visible. Docs in this repo feel buried to me.

**Review by JJediny — COMMENTED** [review]
@rocheller123 this is ready for merging need your approval

**Pull request merged**